frontend/src/Pages/AdminManagement.tsx

```diff
@@ -6,7 +6,7 @@ import {
     ArrowPathRoundedSquareIcon,
     PencilIcon,
     TrashIcon,
-    UserPlusIcon
+    PlusIcon
 } from '@heroicons/react/20/solid';
```

Comment:
@MillerSenior Some of these appear to look too thick in the UI compared to the figma. Can you try using `outline` instead of `solid`??

Reply:
Done